nncf/quantization/quantize_model.py

```diff
@@ -506,20 +510,25 @@ def compress_weights(
             compress_weights_impl as fx_compression_weights_impl,
         )
 
-        if mode not in [CompressWeightsMode.INT8_ASYM, CompressWeightsMode.INT8_SYM]:
+        if mode in [CompressWeightsMode.NF4, CompressWeightsMode.E2M1]:
+            raise AttributeError("Torch backend does not support NF4 and E2M1 modes for weight compression.")
+
+        options = {
+            "sensitivity_metric": sensitivity_metric,
+            "awq": awq,
+            "scale_estimation": scale_estimation,
+            "gptq": gptq,
+            "lora_correction": lora_correction,
+        }
+        unsupported_options = [name for name, value in options.items() if value is not None]
+        if unsupported_options:
             raise AttributeError(
-                "TorchFX backend supports only INT8_ASYM, INT8_SYM modes for weight compression, "
-                f"but given {mode.value} mode."
+                f"TorchFX backend does not support {', '.join(unsupported_options)} option(s). Set them to None."
             )
 
-        if backup_mode is not None:
-            raise AttributeError("TorchFX backend does not support backup_mode option.")
+        if ratio is not None and ratio != 1:
```

Comment:
As far as I can see, Torch and Torch FX have the same processing of parameters. Does it make sense to combine them in a single function?

Reply:
Thanks for this comment. Thanks for this comment. We are going to refactor it when the torch.fx backend will be ready to move out from experiments module.

cc' @daniil-lyakhov, @anzr299